azbfs/zc_policy_retry.go

```diff
@@ -284,7 +283,7 @@ func NewRetryPolicyFactory(o RetryOptions) pipeline.Factory {
 				}
 				if response != nil && response.Response() != nil {
 					// If we're going to retry and we got a previous response, then flush its body to avoid leaking its TCP connection
-					io.Copy(ioutil.Discard, response.Response().Body)
+					_, _ = io.Copy(io.Discard, response.Response().Body)
```

Comment:
i wonder why it chose io over ioutil

Reply:
ioutil is a deprecated package and this is the suggested fix. This was a manual change on my part.